book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] You are supposed to have both Getters and Setters for @rentals
`attr_accessor :rentals`
The getter method allows you to access the value of the @rentals instance variable from outside the class, while the setter method allows you to assign a new value to the @rentals instance variable.

Reply:
Thanks, it's done